guides/leon/vsftpd.md

```diff
@@ -0,0 +1,211 @@
+# VSFTP GUIDE
+
+## Summary 

```

Comment:
Also follow the template, the entire thing will flow with everyone else guide. Summary should be above after Prerequisites.